modules/apps/notification/notification-rest-impl/rest-openapi.yaml

```diff
@@ -71,6 +117,105 @@ info:
     version: v1.0
 openapi: 3.0.1
 paths:
+    "/notification-queue-entry":
```

Comment:
@p-albuquerque This path should be using the plural. Can you rename it? `/notification-queue-entries`